index.html

```diff
@@ -52,8 +52,16 @@
 
 		<!-- ## SLIDE TEXT ## -->
 		<div id="qcSlideText">
-			<h1 id="slidecaption">Write/Speak/Code 2017 conference</h1>
-			<p id="slidedesc">Join us for 4 intense days of workshops, seminars, and talks.<br>Let's build the blueprint that will take your career to the next level!</p>
+			<h1 id="slidecaption"><span class="wsc">Write/Speak/Code</span> 2017 conference</h1>
+			<div id="slidedesc">
```

Comment:
@deweydell The problem was that `Write/Speak/Code` was too long to fit on the (Chrome's simulator's) iPhone 5 screen, and the rest of the text was fine. Do you think it would be better to make all the text smaller?